tests/data/library_of_congress/bd01x09x.xml

```diff
@@ -309,14 +309,12 @@
     <datafield tag="017" ind1=" " ind2=" ">
       <subfield code="a">99-263</subfield>
       <subfield code="b">BwMiBKP </subfield>
-      <subfield code="2"></subfield>
```

Comment:
I am removing them because they are not preserved in the roundtrip MARC -> JSON -> MARC. That doesn't exclude they are perfectly supported in the simple MARC -> JSON transformation.